pkg/cmd/repo/gitignore/view/view.go

```diff
@@ -0,0 +1,101 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type ViewOptions struct {
+	IO         *iostreams.IOStreams
+	HTTPClient func() (*http.Client, error)
+	Config     func() (gh.Config, error)
+	Template   string
+}
+
+func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Command {
+	opts := &ViewOptions{
+		IO:         f.IOStreams,
+		HTTPClient: f.HttpClient,
+		Config:     f.Config,
+		Template:   "",
+	}
+
+	cmd := &cobra.Command{
+		Use:   "view <template>",
+		Short: "View an available repository gitignore template",
+		Long: heredoc.Docf(`
+		View an available repository %[1]s.gitignore%[1]s template.
+		
+		%[1]s<template>%[1]s is a case-sensitive %[1]s.gitignore%[1]s template name.
+		`, "`"),
+		Args: cmdutil.ExactArgs(1, "gh repo gitignore view only takes a single template argument"),
```

Comment:
Again on bringing this into alignment with other commands language

Reply:
Fixed 👍